astrodb_utils/__init__.py

```diff
@@ -1,21 +1,24 @@
-import logging
+import logging  # noqa: I001
 import sys
 import warnings
 
 from .utils import (  # noqa: F401
     AstroDBError,
     exit_function,
-    ingest_instrument,
     internet_connection,
     load_astrodb,
 )
+from .instruments import (  # noqa: F401
+    ingest_instrument,
+)
+
 from .version import version as __version__
 
 __all__ = ["__version__"]
 
 
 
-logger = logging.getLogger("astrodb_utils")

```

Comment:
Double check if you should keep the explicit name of the logger here.

Reply:
I don't have to, but I want to.